nautobot/circuits/templates/circuits/circuit.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'generic/object_detail.html' %}
+{% extends 'generic/object_retrieve.html' %}
```

Comment:
Future cleanup (either as comment in this file or as backlog) to remove the need for these files.